inc/HttpClient/RdbCacheStrategy.php

```diff
@@ -14,7 +14,7 @@
 
 class RdbCacheStrategy extends GreedyCacheStrategy {
 	private const CACHE_INVALIDATING_REQUEST_HEADERS = [ 'Authorization', 'Cache-Control' ];
-	private const FALLBACK_CACHE_TTL_IN_SECONDS = 60;
+	private const FALLBACK_CACHE_TTL_IN_SECONDS = 120;
```

Comment:
Originally, I had this at 10 mins but that felt like a lot. I've made this 2 mins as that accounted for the case where a query could take quite a while to come back, and we may have several of em. It helps subsequent requests. 

Happy to revert this, just found that on a complicated site this helped to delay when subsequent requests could take a while to load due to the cache expiring.

Reply:
> I've made this 2 mins as that accounted for the case where a query could take quite a while to come back, and we may have several of em

I'm not sure I understand this statement. Certainly increasing the default TTL will help make cache misses less frequent (for queries that use the default TTL). Is there another effect I'm missing?

Reply:
Nope, that's really it